variants/tbeam-s3-core/variant.h

```diff
@@ -1,12 +1,18 @@
 // #define BUTTON_NEED_PULLUP // if set we need to turn on the internal CPU pullup during sleep
 
-#define I2C_SDA         42
-#define I2C_SCL         41
+#define I2C_SDA1         42         //Used for PMU management
+#define I2C_SCL1         41         //Used for PMU management
+
+#define I2C_SDA          17         //For sensors and screens
+#define I2C_SCL          18         //For sensors and screens
 
 #define BUTTON_PIN      0     // The middle button GPIO on the T-Beam S3
 //#define BUTTON_PIN_ALT 13 // Alternate GPIO for an external button if needed. Does anyone use this? It is not documented anywhere.
 // #define EXT_NOTIFY_OUT 13 // Default pin to use for Ext Notify Module.
 
+#define BUTTON1_PIN     47      //Additional keys
+
+
```

Comment:
What will the button be used for? If there's no default action it may be good to leave it undefined and map it through the inputbroker.

Reply:
At present, this button is not used. I just want to highlight that there is an additional button in the t-beam-s3-core. Can I add the function in the PR later

Reply:
maybe leave the define in and comment it out? We can map additional pins/buttons through a run time configuration, e.g. for a rotary encoder or external sensors.

Reply:
This cannot be done because on hardware it has been defined as a key (already physically connected as a key) and it cannot be used as another function